lib/SPIRV/SPIRVLowerSaddWithOverflow.h

```diff
@@ -35,22 +35,21 @@
 #ifndef SPIRV_SPIRVLOWERSADDWITHOVERFLOW_H
 #define SPIRV_SPIRVLOWERSADDWITHOVERFLOW_H
 
-#include "llvm/IR/InstVisitor.h"
 #include "llvm/IR/PassManager.h"
 #include "llvm/Pass.h"
 
 namespace SPIRV {
 
-class SPIRVLowerSaddWithOverflowBase
```

Comment:
I guess we should drop the "WithOverflow" part from the class name now (and maybe rename the file too).  But perhaps it would be better to do that in a followup commit/PR.

Reply:
+1

Reply:
The main reason I kept the name was I didn't have a particularly better name for the pass, given that not all of the `*_with_overflow` or `*_sat` intrinsics are lowered here.

Maybe `LowerSaddIntrinsics`?

Reply:
Either `LowerSadd` or `LowerSaddIntrinsics` sound good to me.